pkgs/top-level/all-packages.nix

```diff
@@ -9093,17 +9093,15 @@ let
   neard = callPackage ../servers/neard { };
 
   nginx = callPackage ../servers/http/nginx {
-    rtmp        = true;
-    fullWebDAV  = true;
-    syslog      = false; # the patch is not found
-    moreheaders = true;
+    modules = [ nginxModules.rtmp nginxModules.dav nginxModules.moreheaders ];
```

Comment:
I was afraid that it would clash with something in `pkgs` -- if I remember correctly `rec` would take precedence over `with` statement (yuck, dark parts of Nix) and names seem general enough to overlap with some package name.